src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -60,10 +60,13 @@
 	Throwable inboundError;
 
 	volatile Disposable receiverCancel;
-	volatile int wip;
 
-	final static AtomicIntegerFieldUpdater<FluxReceive> WIP = AtomicIntegerFieldUpdater.newUpdater
-			(FluxReceive.class, "wip");
+	volatile int once;
+	static final AtomicIntegerFieldUpdater<FluxReceive> ONCE =
+		AtomicIntegerFieldUpdater.newUpdater(FluxReceive.class, "once");
+
+	int wip;
```

Comment:
Nitpick: I would just add the same kind of comment than in this PR's body next to `wip`, or in the class' javadoc (ie. why there's no need for volatile `wip` nor MPSC queues, due to running on event loop)

Reply:
Sure

Reply:
✅